lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
When there is a multi-line flex container, this doesn't work as expected on the last item of the non-last line(s).  I suggest removing the margin from the cards and using `gap` instead.

Reply:
> When there is a multi-line flex container, this doesn't work as expected on the last item of the non-last line(s). I suggest removing the margin from the cards and using `column-gap` instead.

https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/14e55445ecb4714a15553f7625a7549a8c9efe1a